docs/source/release/v6.3.0/muon.rst

```diff
@@ -85,35 +75,32 @@ Improvements
 
 - :ref:`LoadPSIMuonBin <algm-LoadPSIMuonBin>` can now load a subset of the spectra.
 
-Bugfixes
-########
-
 
 Fitting Functions
 -----------------
 New Features
 ############
-* Added two Activation fitting functions to MuonModelling Fit Functions.
+* Added two Activation fitting functions to ``MuonModelling`` Fit Functions.
 
-  * `ActivationK <func-ActivationK>` can be used for data in Kelvins.
-  * `ActivationmeV <func-ActivationmeV>`can be used for data in meV.
+  * :ref:`ActivationK <func-ActivationK>` can be used for data in Kelvins.
+  * :ref:`ActivationmeV <func-ActivationmeV>`can be used for data in meV.
 
-* Added a :ref:`Critical peak of relaxation rate <func-CriticalPeakRelaxationRate>` for fitting to MuonModelling\Magnetism Fit Functions.
+* Added a :ref:`Critical peak of relaxation rate <func-CriticalPeakRelaxationRate>` for fitting to ``MuonModelling\Magnetism`` Fit Functions.
 * Added two fitting functions for the decoupling of asymmetry in the ordered state of a powedered magnet for fitting.
 
-  * `DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
-  * `DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
+  * :ref:`DecoupAsymPowderMagLong <func-DecoupAsymPowderMagLong>` can be used for longitudinal polarization.
+  * :ref:`DecoupAsymPowderMagRot <func-DecoupAsymPowderMagRot>`can be used for rotational asymmetry.
```

Comment:
This link does not work

Reply:
Again is this both links or just the second one?

Reply:
the second in both cases

Reply:
I think its just a formatting problem

Reply:
Yeah its because there is no space after the ` at the end of the link and the next word.